website/docs/basic_configurations.md

```diff
@@ -74,7 +74,7 @@ Options useful for writing tables via `write.format.option(...)`
 > Record key field. Value to be used as the `recordKey` component of `HoodieKey`.
 Actual value will be obtained by invoking .toString() on the field value. Nested fields can be specified using
 the dot notation eg: `a.b.c`<br></br>
-> **Default Value**: uuid (Optional)<br></br>
+> **Default Value**:  N/A (Required)<br></br>
 > `Config Param: RECORDKEY_FIELD`<br></br>
```

Comment:
The default primary key `uuid` is broken now?

Reply:
Sorry for this incorrect change, this configuration has been changed in Flink and default value 'uuid' was removed since this [pr-6681](https://github.com/apache/hudi/pull/6681) in Spark.

Reply:
Can we fix the other version also?

Reply:
I have checked again, Default value change of 'hoodie.datasource.write.recordkey.field' was introduced only in lately [release-0.13.0](https://github.com/apache/hudi/blob/release-0.13.0/hudi-common/src/main/java/org/apache/hudi/keygen/constant/KeyGeneratorOptions.java) and wasn't changed in [release-0.12.2](https://github.com/apache/hudi/blob/release-0.12.2/hudi-common/src/main/java/org/apache/hudi/keygen/constant/KeyGeneratorOptions.java).  
so no more versions to be fixed.